docs/platform/concepts/beta_services.rst

```diff
@@ -1,13 +1,13 @@
 Service and feature releases
 =============================
 
-Before general availability, the lifecycle of new services and features includes a limited availability stage and an early availability stage.
+Before general availability, the lifecycle of new services and features includes a limited availability (beta) stage and an early availability stage.
```

Comment:
limited availability (private beta) stage and an early availability (public beta) stage.